tests/features/targets/import_targets.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Import targets directly to RSTUF SQL Database
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+    Admin has an existent large number of targets (i.e.: 1,000,000 targets)
+    Admin wants to load targets in chunks to process them faster
+
+    Background:
+        Given admin has the RSTUF CLI in a host with access to RSTUF SQL Database
+        And admin has logged in with RSTUF CLI
+        And admin has the credentials to access RSTUF SQL Database
+        And admin has exported the targets info to CSV files (respecting CSV limits) with the format 'path;size;hash_type;hash;custom;'
+
+    Scenario: Admin imports a large number of targets to RSTUF
+        Given admin has multiple CSV files with targets 'tests/data/targets-1of2.csv' and 'tests/data/targets-2of2.csv'
+        When admin runs 'rstuf -c rstuf.ini admin import-targets -csv tests/data/targets-1of2.csv -csv tests/data/targets-2of2.csv -metadata-url http://127.0.0.1:8080 -db-uri postgresql://postgres:secret@127.0.0.1:543'
+        And the admin expects to get confirmation 'CSV data was imported successfully'.
+        And the admin expects to get the task id in the message 'Publishing new targets in task id:'
```

Comment:
Does the publishing operation take most of the time?
How long does it take to finish the step on line 16? Do we need to give a task id that could be traced for step on line 16?

Reply:
The steps time [are these](https://github.com/vmware/repository-service-tuf/issues/188#issuecomment-1400576033).
To summarize:
- Import 500,000 ~15 seconds (line 16)
- Publishing targets: 500,000 40 minutes, 1M 55 minutes (line 17)

The task id is needed because, automatically, the CLI will keep the user updated about the task. If for some reason, the CLI session is interrupted, the user can use the task id to see the task status.